### DISCUSSION TITLE
Changing playable characters (FNF 0.5.2)

### DISCUSSION THREAD

**Discussion by ghost** [FunkinCrew/Funkin#3815] [Q&A] [answered]
I couldn't find any option for changing the characters, i want to change my playable character to Pico, that i've just unlocked. If there's no options, then it could be any of keys/key combinations.

**Comment by amyspark-ng** [comment, marked as answer]
im pretty sure freeplay will show a text that says "Tab to change character" or something like that once you have a new available character to play as

Also options > controls > freeplay pretty sure as an option for tab for char select, maybe you're playing the wrong version?
